client/my-sites/site-settings/spam-filtering-settings.jsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { includes } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import FormInputValidation from 'components/forms/form-input-validation';
+import Gridicon from 'gridicons';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getJetpackSettingsSaveError, getJetpackSettingsSaveRequestStatus } from 'state/selectors';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+
+const SpamFilteringSettings = ( {
+	currentAkismetKey,
+	dirtyFields,
+	fields,
+	hasAkismetKeyError,
+	isRequestingSettings,
+	isSavingSettings,
+	onChangeField,
+	translate,
+} ) => {
+	const { akismet: akismetActive, wordpress_api_key } = fields;
+	const isStoredKey = wordpress_api_key === currentAkismetKey;
+	const isDirty = includes( dirtyFields, 'wordpress_api_key' );
+	const inTransition = isRequestingSettings || isSavingSettings;
+	const isValidKey =
+		( wordpress_api_key && isStoredKey ) ||
+		( wordpress_api_key && isDirty && isStoredKey && ! hasAkismetKeyError );
+	const isInvalidKey = isDirty && hasAkismetKeyError && ! isStoredKey;
+	let validationText, className, header = null;
+	if ( ! inTransition && isValidKey ) {
+		validationText = translate( 'Your Antispam key is valid.' );
+		className = 'is-valid';
+		header = (
+			<div>
+				<Gridicon icon="checkmark" />
+				{ translate( 'Your site is protected from spam.' ) }
+			</div>
+		);
+	}
+	if ( ! inTransition && isInvalidKey ) {
+		validationText = translate( "There's a problem with your Antispam API key." );
+		className = 'is-error';
+		header = (
+			<div>
+				<Gridicon icon="notice-outline" />
+				{ translate( 'Your site needs an Antispam key.' ) }
+			</div>
+		);
+	}
+	return (
+		<FoldableCard
+			header={ header }
+			className="spam-filtering__foldable-card site-settings__foldable-card"
+		>
+			<FormFieldset>
+				<div className="spam-filtering__settings site-settings__child-settings">
+					<div className="spam-filtering__info-link-container site-settings__info-link-container">
+						<InfoPopover>
+							<ExternalLink
+								target="_blank"
+								icon
+								href={ 'https://jetpack.com/features/security/spam-filtering/' }
+							>
+								{ translate( 'Learn more about spam filtering.' ) }
+							</ExternalLink>
+						</InfoPopover>
+					</div>
+					<FormLabel htmlFor="wordpress_api_key">{ translate( 'Your API Key' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **42** times:
`translate( 'API Key' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/akismet-card-for-jetpack-sites-in-settings-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).